intelmq/lib/bot.py

```diff
@@ -245,7 +265,11 @@ def acknowledge_message(self):
         self.last_message = self.current_message
         self.source_pipeline.acknowledge()
 
-    def dump_message(self):
+    def dump_message(self, error_traceback):
+        if self.current_message is None:
```

Comment:
hum... ok. If you, with your user experience felt that, I think we can change it, no problem. hum...Im wondering how we should limit the number of equal dumps into dump file? The scenario is:
1. broker fails
2. `error_procedure=retry`
3. `error_dump_message=true`
4. `error_retry_delay=15`
5. `error_max_retries=2`

Bot will dump the same null message with the same traceback every 30 seconds. In your opinion, how should bot proceed in this situation?